content/02.introduction.md

```diff
@@ -2,5 +2,73 @@
+We discuss previous work done for each type of edge (relationship) studied in this project.
+
+#### Disease-Gene Associations 
+Regarding rule based approaches, Liu et al. used a dictionary lookup approach to extract multiple edges involving diseases, genes, toxins, etc. from literature [@tag:polysearch]. 
+Song et al., created a text mining tool called PKDE4J which uses a dictionary and rule based approaches to extract associations from literature [@tag:pkde4j].
+Despite the great performance, these approach takes significant amount of manual effort to use.  
+In the context of unsupervised approaches, Pletscher-Frankild et al., used a co-occurence approach to predict disease-gene associations [@tag:diseases].
+Each disease and gene pair were ranked based on their mention frequency within in biomedical abstracts.
+Junge et al., used an advanced version of the DISEASES approach that combines co-occurence scores with a distant supervision approach to identify disease gene associations [@tag:cocoscore].
+Kim et al., used a co-occurence scores to construct a disease-gene network to find associations [@tag:lgscore].
+Zhou et al., combined term weights with co-occurence methods to improve performance on association identification [@tag:dg_text_pubmed].
+Westergaard et al., used a co-occurence approach using full text to detect associations [@tag:full_text_co_abstracts].
+Besides co-occurence, Percha et al., used a bi-clustering approach on dependency trees generated from MEDLINE abstracts to detect associations [@tag:global_network].
+These kind of approaches have powerful recall performance, but can suffer precision wise.  
+With the availability of hand annotated datasets [@tag:befree; @tag:eu_adr; @tag:comagc], there have been a number of supervised machine learning approaches for classifying associations.
+Quan et al., used a maximum entropy classifier to classify sentences that indicate an association [@tag:dg_extract_txt_network].
+Bravo et al., used a support vector machine classifier with two different kernels, a dependency kernel and a shallow linguistic kernel, to classify associations [@tag:befree]. 
+Xu and Zhang et al., used a support vector machine classifier with local and global context features to classify associations in literature [@tag:dtminer].
+Recently Bhasuran et al., created used an ensemble approach that consisted of using multiple support vector machines to identify associations [@tag:ensemble_svm].
+Lee et al., build off of the BERT [@tag:bert] system to build a deep learning classifier for association detection [@tag:biobert].
```

Comment:
I thought you were going to say something here about these having better precision - do they? I though the bit about recall was going to come back to the part above. Can you synthesize the literature a bit more? Which things are similar to each other? Which are the most similar to the generative model you are building? Which are the most similar to the discriminative model?

This context is really important for a good related work section and it's missing here (and in the subsections that follow). Can you add it and then I can re-review?